app-accessibility/pipertts/pipertts-1.0.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="An offline text-to-speech program using the NCNN library"
+DESCRIPTION="Piper TTS is an offline text-to-speech program using the NCNN library \
+to provide natural, Human-like voices. Different voice models can be downloaded from \
+https://huggingface.co/gyroing/PiperTTS-NCNN-Models/tree/main."
+HOMEPAGE="https://github.com/gyroing/PiperTTS-HaikuOS/"
+COPYRIGHT="2025 GYROING"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+srcGitRev="abd4330345769f2ceda1592612931aa64a711e5c"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="cffafc6d1ea3a013f3117554b29a7913c7c1894407387bfa497f6606bdc5d264"
+SOURCE_DIR="PiperTTS-HaikuOS-$srcGitRev"
+srcGitRev_2="d7df277f9cc15a00b19877ab75b287ade42a22ab"
+SOURCE_URI_2="https://github.com/Tencent/ncnn/archive/$srcGitRev_2.tar.gz"
+CHECKSUM_SHA256_2="989bb0f7b8a00c24b3b297cdc7f23c6932a5676731c7dda24ea29dfc8eae4668"
+SOURCE_DIR_2="ncnn-$srcGitRev_2"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	pipertts = $portVersion
+	cmd:pipertts = $portVersion
+	"
+REQUIRES="
+	haiku
+	cmd:espeak
+	devel:libgomp
```

Comment:
```suggestion
	lib:libgomp
```

Reply:
If you build with "haikuporter -S" you would see an "POLICY ERROR" otherwise without -S (strict mode) this turns into a POLICY_WARNING, it's what the buildmasters use, so it doesn't error when packaging.
But better resolve these warnings upfront (will need to do a rebuild to test this, saw it on previous build here).